src/components/Message/Message.js

```diff
@@ -0,0 +1,12 @@
+import styles from "./Message.module.css";
+
+
+export const Message = ({ content, time, status }) => {
+    const { send, received, timeStampStyle, message } = styles;
```

Comment:
[opinionated][optional] I prefer not to destructurize styles property and instead of it use just `s` character. 
``
```suggestion
    import s from "./Message.module.css";


export const Message = ({ content, time, status }) => (
      <div className={[status === 'received' ? s.received : s.send, s.message].join(' ')}>
        <p >{content}</p>
        <p className={s.timeStampStyle}>{time}</p>
      </div>
    );
```